tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -185,6 +186,22 @@ static PyObject * THPVariable_detach_(PyObject* self, PyObject* args)
   END_HANDLE_TH_ERRORS
 }
 
+static Tensor dispatch_to_backend(const Tensor & self, Backend backend) {
+  AutoNoGIL no_gil;
+  AutoGPU auto_gpu(self);
+  Tensor t = self.toBackend(backend);
```

Comment:
`return self.toBackend(backend)`?

Reply:
ah ya, this is just some debugging artifact.